web-local/src/lib/svelte-query/queries/metrics-views/metadata.ts

```diff
@@ -58,7 +45,7 @@ export const useMetaMappedFilters = (
   filters: MetricsViewRequestFilter,
   dimensionName?: string
 ) =>
-  useCatalogQuery<MetricsViewRequestFilter>(instanceId, metricViewName, (_) => {
+  useMetaQuery<MetricsViewRequestFilter>(instanceId, metricViewName, (_) => {
```

Comment:
Do we need this query? We are not using the meta query response here right? `useMetaMappedFilters` can just be a function?

Reply:
Good point. I have no idea why this was a query in the 1st place.